spec/queries/find_many_by_string_property_spec.rb

```diff
@@ -0,0 +1,26 @@
+
+# frozen_string_literal: true
+require "rails_helper"
+
+RSpec.describe FindManyByStringProperty do
+  subject(:query) { described_class.new(query_service: query_service) }
+  let(:box) { FactoryBot.create_for_repository(:ephemera_folder, barcode: "1234567") }
+  let(:box2) { FactoryBot.create_for_repository(:ephemera_folder, barcode: "2345678") }
+  let(:query_service) { Valkyrie.config.metadata_adapter.query_service }
+
+  describe "#find_many_by_string_property" do
+    it "can find objects with strings in it by a property" do
+      output = query.find_many_by_string_property(property: :barcode, values: [box.barcode.first, box2.barcode.first])
+      expect(output.length).to eq 2
+      expect(output.first.id).to eq box.id
+      expect(output.last.id).to eq box2.id
```

Comment:
Are these always going to be returned in order?  Or should we make this order-neutral to avoid failures if they come back in the other other?